check_diff/tests/git.rs

```diff
@@ -0,0 +1,50 @@
+use check_diff::{change_directory_to_path, clone_git_repo};
+use std::env;
+use std::ffi::OsStr;
+use std::fs;
+use std::path::Path;
+
+const TEMP_DIR_PATH: &str = "./tmp/";
+const HOME_PATH: &str = "./";
+
+// removes the previous tmp folder if it exists
+fn cleanup() {
+    let _ = env::set_current_dir(HOME_PATH);
+    println!(
+        "Current directory: {}",
+        env::current_dir().unwrap().display()
+    );
+    let path = Path::new(TEMP_DIR_PATH);
+    println!("{}", path.exists());
+    if path.exists() {
+        let _ = fs::remove_dir_all(path);
+    }
+}
+
+#[test]
+fn clone_repo_test() {
```

Comment:
@ytmimi for this test fails when i run `cargo test`. however, when running it on its own, it works fine. For some reason after `clone_git_repo` is called, the current directory becomes `./tmp/`. This is quite unexpected since i did not change directory in `clone_git_repo`.